### PR TITLE
dep(fmt): update ethers & solang

### PR DESCRIPTION
**Pull request by rkrasiuk**

### CODE REVIEW

**Comment by rkrasiuk** [comment]
@mattsse still a draft as i want to check some minor issues

File: fmt/src/formatter.rs [hunk 1]
```diff
@@ -1825,25 +1860,20 @@ impl<'a, W: Write> Visitor for Formatter<'a, W> {
...
-                        helpers::namespace_matches(&contract_base.name, &base.name)
+                        contract_base
+                            .name
+                            .identifiers
+                            .iter()
+                            .zip(&base.name.identifiers)
+                            .all(|(l, r)| l.name == r.name)
                     })
                 });
+                if base.name.identifiers.iter().any(|i| i.name == "Ownable") {}
```
Comment (jpopesculian) [inline fmt/src/formatter.rs:1876]: is this left over from something?

File: fmt/src/formatter.rs [hunk 2]
```diff
@@ -1646,6 +1666,21 @@ impl<'a, W: Write> Visitor for Formatter<'a, W> {
         Ok(())
     }
 
+    fn visit_ident_path(&mut self, idents: &mut IdentifierPath) -> Result<(), Self::Error> {
```
Comment (jpopesculian) [inline fmt/src/formatter.rs:1669]: I think you may always be able to group these if you wanted

**Review by jpopesculian — COMMENTED** [review]
lgtm! just little stuff

File: fmt/src/formatter.rs (hunk 1 above)
Comment (rkrasiuk) [reply fmt/src/formatter.rs:1876]: it is, good eye

**Review by mattsse — APPROVED** [review]

**Pull request merged**